### ISSUE TITLE
[REQ] Sort by Release date

### ISSUE THREAD

**Issue by LordRofl** [Fledge68/WiiFlow_Lite#265]
A new sort option by Release date would be useful and a good addition.

**Comment by Fledge68** [comment]
i really wanted to add this to wiiflow but its turning out to be extremely difficult since the release date is taken from gametdb.xml or the plugin platform database xml. i could add it to games header info but this means everyone upgrading to wiiflow v.5.5.0 and later will have to delete the cached list or wiiflow will crash.

**Comment by eku** [comment]
> upgrading to wiiflow v.5.5.0 and later will have to delete the cached list or wiiflow will crash

Sounds like the cache has no version info inside and wiiflow does not know which version has build it. We should change this while we implement new sort options. So wiiflow can take care of imcompatible format in the future.

**Comment by Fledge68** [comment]
yes there's no header info included with each cached list file. back when wiiflow was on googles svn site fix94 had wiiflow delete the lists folder any time a new svn revision was started but i removed that when it was moved to github.

i plan on adding to wiiflow_lite.ini under [GENERAL] cached_list_version=550. and on every boot wiiflow would compare this to the current version. if the key was not found or if the cached list version is < the current version then wiiflow will delete the lists folder to force wiiflow to recreate the lists.

of course every time we update/change the game header (dir_discHdr) like adding a sort option we must also update the current version value.

but i don't want to add too much to struct dir_discHdr because that will increase the size of the lists and if a user has a lot of games it will consume a lot of memory.
